debug_toolbar/static/debug_toolbar/js/redirect.js

```diff
@@ -0,0 +1,3 @@
+document.addEventListener('DOMContentLoaded', function () {
+  document.getElementById('redirect_to').focus();
```

Comment:
https://caniuse.com/#search=autofocus

Reply:
This is focussing a link not a form element :(